### DISCUSSION TITLE
Upgrade from 7 to 8

### DISCUSSION THREAD

**Discussion by AnishAppanath** [AmadeusITGroup/otter#1613] [Q&A] [answered]
I am trying to upgrade from 7 to 8 with the steps mentioned in your migration guide. But facing the below issue in the global theming. The global theme in the application is not recognized by the component.

<img width="304" alt="Screenshot 2024-04-10" src="https://github.com/AmadeusITGroup/otter/assets/20016933/13351ae7-2391-49e8-95d6-72a3c7178ce6">

**Comment by kpanot** [comment, marked as answer]
Hello @AnishAppanath ,

I closing you question because it is not ask at the right place.
Please be aware that this Q&A section is dedicated to the packages part of this repository and the first version of the packages part of this repository is the version 8.0.
